docs/pages/project/community.md

```diff
@@ -8,7 +8,7 @@ language: en
 type: project
 ---
 
-Meshery is a CNCF prpoject maintained by engineers from Layer5, VMware, Red Hat, Rackspace, Citrix, Layer5, Instabase, Microsoft, OpenGov, Computas AS, Rill Data, Quantex, Lumina Networks, and HashiCorp. Meshery is an an open source project that adheres to the [CNCF's Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Meshery advance the state of the art surrounding service meshes and enable adopters of Cloud Native technologies and techniques to do so with confidence. Our shared commitment to the open source spirit pushes Meshery forward.
+Meshery is a CNCF project maintained by engineers from Layer5, VMware, Red Hat, Rackspace, Citrix, Layer5, Instabase, Microsoft, OpenGov, Computas AS, Rill Data, Quantex, Lumina Networks, and HashiCorp. Meshery is an open source project that adheres to the [CNCF's Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Meshery advance the state of the art surrounding service meshes and enable adopters of Cloud Native technologies and techniques to do so with confidence. Our shared commitment to the open source spirit pushes Meshery forward.
 
 #### Community First
 
```

Comment:
```suggestion
<p>Meshery is one of a number of open source projects originated by the <a href="https://layer5.io">Layer5 community</a>, which represents the largest collection of service mesh projects and their maintainers in the world.</p>
```
I'm unable to suggest these changes for line number 15. Please check.

Reply:
What is the change here? 🤔

Reply:
orginated => originated

Reply:
Aah...ok.
Maybe another PR?

Reply:
OK, I'll create one.